src/layout/section/mediaContentSection/MediaContentSection.story.mdx

```diff
@@ -31,5 +30,17 @@ stories.add('default', () => (
       buttonHref="https://www.blablacar.fr/bus"
       mediaUrl="https://cdn.blablacar.com/kairos/assets/build/images/blablabus_bus_logo_large-4717a333c3a2184a5849b0cbb6da75cb.png"
     />
-  </Fragment>
-))
+  </Story>
+</Preview>
+
+## Specifications
+
+## Usage
+
+```jsx
+import { MediaContentSection } from '@blablacar/ui-library/build/MediaContentSection'
```

Comment:
```suggestion
import { MediaContentSection } from '@blablacar/ui-library/build/mediaContentSection'
```

Reply:
Actually it's `import { MediaContentSection } from '@blablacar/ui-library/build/layout/section/mediaContentSection'`